src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Attachments/index.test.js

```diff
@@ -1,10 +1,59 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
 
-// import Attachments from 'index';
+import Attachments from './index';
```

Comment:
Should be `import Attachments from './';`

Reply:
idem